pkg/daemon/sync.go

```diff
@@ -70,6 +70,8 @@ func (d *Daemon) Sync(ctx context.Context, started time.Time, newRevision string
 		return err
 	}
 
+	d.Logger.Log(fmt.Sprintf("Trying to sync git changes to cluster, from commit %s to commit %s", c.oldTagRev, c.newTagRev))
```

Comment:
```suggestion
	d.Logger.Log(fmt.Sprintf("Trying to sync git changes to cluster, from commit '%s' to commit '%s'", c.oldTagRev, c.newTagRev))
```

Would be easier to read in a blob of logs imo.